### PR TITLE
API view cleanup

### PR DESCRIPTION
**Pull request by brianparry**
* Remove references to injector from views.
* use 'renderable' view for profiles, templates, and views.
* cleanup nodes.obms and obms view.

@RackHD/corecommitters @keedya @dalebremner @BillyAbildgaard

### CODE REVIEW

**Comment by yyscamper** [comment]
:+1:

**Comment by keedya** [comment]
:+1:

**Comment by keedya** [comment]
test this please

File: data/views/obm.2.0.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
+  <% if (node !== null) { %>
+  "node": "<%=basepath%>/nodes/<%=node%>",
+  <% } %>
   "service": "<%= service %>",
   "config": <%- JSON.stringify(config) %>
 }
```
Comment (dalebremner) [inline data/views/obm.2.0.json]: As we discussed, we may need to display the OBM id in the view too...

**Comment by benbp** [comment]
👍

**Comment by brianparry** [comment]
depends on https://github.com/RackHD/RackHD/pull/321

File: data/views/obm.2.0.json (hunk 1 above)
Comment (brianparry) [inline data/views/obm.2.0.json]: @dalebremner Fixed.

**Comment by brianparry** [comment]
test this please

**Pull request merged**